### PR TITLE
Add a print() function to the JIT script

### PR DESCRIPTION
**Pull request by apaszke**
Additionally:
- add support for calling functions that are not methods in the Python frontend
- add an end-to-end test for the Python frontend
- add a capture_stdout helper for checking that `print` actually works

### CODE REVIEW

File: torch/csrc/jit/script/compiler.cpp [hunk 1]
```diff
@@ -471,13 +471,21 @@ struct to_ir {
         const auto& inputs = tree->trees();
         auto kind = getNodeKind(tree->kind(), inputs.size());
         auto* node = emitNode(kind, getValues(inputs), output_size);
-        node->t_(Symbol("alpha"), at::CPU(at::kFloat).scalarTensor(1.0));
+        if (inputs.size() == 2) {
```
Comment (zdevito) [inline torch/csrc/jit/script/compiler.cpp]: I think another PR fixed this.

**Review by zdevito — APPROVED** [review]
This looks fine to rebase/merge once control flow goes int.

**Comment by apaszke** [comment]
Rebased. Should be good to merge once the CI will be up and green.

**Pull request merged**